app/models/user.rb

```diff
@@ -61,4 +60,8 @@ def aasm_ensure_initial_state
   def self.generate_and_send_magic_link(email)
     MagicLink::GenerateAndSendService.call(email: email, site: site)
   end
+
+  def accepted_terms?
```

Comment:
Is there test? Should this be `not_accepted_terms?` instead?

Reply:
Good spot, a direct test appears to be missing, we will add. As for naming I think not including the negation helps with comprehension. Alternatively we can also implement `not_accepted_terms?` for convenience

Reply:
Test has now been added